### ISSUE TITLE
Google Analytics & Yandex Metrika forms tracking

### ISSUE THREAD

**Issue by parfilov** [pojome/elementor#2685]
Hi! Elementor is becoming more popular among marketers and we need more analytics functionality. Please add the new toggle into Forms widget that would appear after select options after submit, for example, the options names would be Track GA and Track YM. When selected we have one (or two) more toggle with options.

**Fields for Google Analytics:**

- Category;
- Action;
- Label;
- Price.

(Names may vary because translating from Russian GA)

**For Yandex Metrika we need 2 fields:**

- Counter number (gives by the Yandex Metrika service when creating site counter). Only digits.
- goal ID

Here is the code examle for track form buttons and thats all we need to track forms with Yandex Metrika:

```
<form action="" method="get" onsubmit="yaCounterXXXXXX.reachGoal('ORDER', function () {
    alert('Data successfully sent');
}, <Context available for the "this" keyword inside the callback>); return true;">
        ...
</form>
```
where
XXXXXX — counter number
ORDER — goal ID

Here is [the manual](https://yandex.com/support/metrica/reports/add-goals.xml) for working with Yandex Metrika goals.

With this scenario we could track every form separately for better understanding which is converting better.

Thanks!

**Comment by bainternet** [comment]
I merged this issue with  #2568 , which are in the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!

**Comment by ramiy** [comment]
This feature can be implemented with the new **Element Attributes** planned for Elementor 2.1 (see PR #4030).
